src/pages/privacy-policy.md

```diff
@@ -0,0 +1,165 @@
+
+We do not retain information about your transaction history, but you acknowledge that the smart contract transactions facilitated by the Network are recorded in perpetuity on the Ethereum blockchain and cannot be reversed or erased. By using the Network and its smart contracts, you acknowledge that the posting of information concerning your Ethereum wallet address is a function of the smart contract functionality to which you agree.
+
+## 4. Children&#39;s data <a id="ChildrensOnlinePrivacy"></a>
+
+We do not direct our websites to minors, and we do not knowingly collect personal information from children under 18. If you are under 18, do not use or provide any information on this website. If we learn we have collected or received personal information from a child under 18 without verification of parental consent, we will delete that information. If you believe we might have any information from or about a child under 18 please contact us using our Contact Us page.
+
+## 5. How we collect your information <a id="HowWeCollectInfo"></a>
+
+We use different methods to collect information from and about you including through:
+
+- **Direct interactions.** You may give us information about you by interacting with our smart contracts or by communicating with us e-mail, or otherwise. This includes information you provide when you use our Network, subscribe to one of our newsletters or other communications, join a user group, complete a survey, or when you contact us about one of our products, or services.
+- **Third parties or publicly available sources.** We may receive information about you from third parties including, for example, our affiliated companies, business partners, sub-contractors, analytics providers, and service providers.
+- **Technical Data from the following parties:**
+  - Analytics providers such as Crowdcast.
+
+## 6. Cookies and automatic data collection technologies <a id="CookiesAndAutoDataCollection"></a>
+
+Our websites may use automatic data collection technologies to distinguish you from other website users. This helps us deliver a better and more personalized service when you browse our website. It also allows us to improve our websites by enabling us to:
+
+- Estimate our audience size and usage patterns.
+- Store your preferences so we may customize our websites according to your individual interests.
+- Recognize you when you return to our website.
+
+The technologies we use for this automatic data collection may include:
+
+- **Cookies (or browser cookies).**A cookie is a small file placed on the hard drive of your computer. For information about managing browser settings to refuse cookies, see _[Your Rights and Choices](#YourCaliforniaPrivacyRights)_.
```

Comment:
The lack of space before “A cookie” here is leading to some GitHub syntax highlighting issues. Not a huge deal as the final rendering looks right (although a little crowded), but flagging it.